src/products/AddProduct.tsx

```diff
@@ -0,0 +1,9 @@
+const AddProduct = (props: any) => {
```

Comment:
Create interface for AddProduct props. Avoid “any”

Reply:
IAddProduct interface has been created in order to avoid any.